samples/esb/esb_prx/sample.yaml

```diff
@@ -33,6 +33,8 @@ tests:
       - nrf54l15dk/nrf54l05/cpuapp
       - nrf54l15dk/nrf54l10/cpuapp
       - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54lm20dk/nrf54lm20a/cpuapp
+      - nrf54lm20pdk/nrf54lm20a/cpuapp
```

Comment:
Are we supposed to have both dk and pdk target?

Reply:
I was wondering the same. Seems like currently other samples have both supported: e.g. https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bluetooth/peripheral_uart/sample.yaml#L19 so let's keep it like this I think, until communicated otherwise.